edgedb-init.sh

```diff
@@ -173,7 +173,7 @@ get_architecture() {
     esac
 
     if [ "$_cputype" = "arm64" ]; then
-        if [ "$_ostype" = "macos" ]; then
+        if [ "$_ostype" = "apple-darwin" ]; then
             # Rely on Rosetta for now
```

Comment:
Can we print here that rosetta will be used? Maybe even print that "softwareupdate --install-rosetta --agree-to-license" can be used to manually install rosetta 2?

Reply:
What does macOS print when you try to run an x86 binary?  It seems to me that this should be automatic on macOS side of things, no?